clippy_dev/src/lintcheck.rs

```diff
@@ -116,9 +115,21 @@ impl CrateSource {
                 // url to download the crate from crates.io
                 let url = format!("https://crates.io/api/v1/crates/{}/{}/download", name, version);
                 println!("Downloading and extracting {} {} from {}", name, version, url);
-                let _ = std::fs::create_dir("target/lintcheck/");
-                let _ = std::fs::create_dir(&krate_download_dir);
-                let _ = std::fs::create_dir(&extract_dir);
+                std::fs::create_dir("target/lintcheck/").unwrap_or_else(|err| {
+                    if err.kind() != ErrorKind::AlreadyExists {
```

Comment:
I changed it to only allow the dir to exist. I think panicking for other things should be fine?

Reply:
Yeah, seems fine. :)